Dockerfile.ci

```diff
@@ -0,0 +1,17 @@
+ARG commit=latest
+FROM elifesciences/proofreader-php:latest AS proofreader
+FROM elifesciences/api-dummy:${commit}
+ENV DEBUG=1
+
+USER elife
+COPY --from=proofreader --chown=elife:elife /srv/proofreader-php /srv/proofreader-php
+RUN ln -s /srv/proofreader-php/bin/proofreader /srv/bin/proofreader
```

Comment:
Is this not a reusable thing already?

Reply:
The `elifesciences/proofreader-php` image contains everything pre-built, so the minimal step here is to copy the `/srv/proofreader-php` folder from there. Then a symlink is useful to just use `proofreader ...` as a command.

Possible further extractions could be:
- `COPY` the symlink and substitute the `RUN` step with it (1 line per 1 line)
- `RUN` some script instead that wraps `ln -s` (1 line per 1 line)